tshirts.c

```diff
@@ -13,10 +13,13 @@ char size(int cms) {
     return sizeName;
 }
 
-int main() {
+int main() 
+{
     assert(size(37) == 'S');

```

Comment:
38 and 42 are boundary values and must have test cases. The test cases for boundary values clearly asserts the expectations from the production code.